app.rb

```diff
@@ -0,0 +1,97 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rentals'
+require_relative 'get_input'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize(books = [], people = [], rentals = [])
+    @books = books
+    @people = people
+    @rentals = rentals
+  end
+
+  def list_all_books
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def list_all_people
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]:'
+    input = gets.chomp
+
+    case input
+    when '1' then create_student
+    when '2' then create_teacher
+    else
```

Comment:
- [ ] I can't start the application, There's no starting point of the app. Please make sure to add a `main.rb` file that will be the entry point of your app and place the list of options in the file and call the methods from the `app.rb` file based on user selection.

![image](https://user-images.githubusercontent.com/83803414/199729437-cecde964-3364-4a7e-bc17-af81bd2c0e07.png)